ast/edit/task_editor.go

```diff
@@ -0,0 +1,127 @@
+package edit
+
+import (
+	"fmt"
+
+	"github.com/influxdata/flux"
+	"github.com/influxdata/flux/ast"
+	"github.com/influxdata/flux/codes"
+)
+
+// GetOption finds and returns the AST node for the option value.
+func GetOption(file *ast.File, name string) (ast.Node, error) {
```

Comment:
The typescript API returns an Expression here instead of a Node.

Reply:
@jlapacik is the expression returned the VariableAssignment node or the Init expression?

Reply:
It's the `init` expression as VariableAssignment is of type `Statement`

Reply:
Updated this as well.